packages/storage-driver-azure/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I very much welcome the fact that you're using the recommended config from https://github.com/tsconfig/bases/blob/main/bases/node18-strictest-esm.combined.json.
WDYT about going a step further by installing `@tsconfig/node18-strictest-esm` and extend the configs from this package?
```json
"extends": "@tsconfig/node18-strictest-esm/tsconfig.json"
```
That way things would be much clearer (as we can easily spot additional settings) and switching to a newer / other version would be a smoother story.

I know it's a bit off-topic but it would be a good time to do so now since this pull request affects 7 `tsconfig.json` files 😃
(Side note: This was also something I wanted to address in #9375 but wasn't aware of [tsconfig/bases](https://github.com/tsconfig/bases) at this time)

Also a bit finicky but you can omit the `exclude` property here since it defaults to "node_modules" and `outDir` anyway (see https://typescript-v2-121.ortam.vercel.app/docs/handbook/tsconfig-json.html#details).

Reply:
Oh lovely! Didn't realize this config also existed as a pre-packaged thing 🙂 I used https://gist.github.com/rijkvanzanten/be696ff0d267b17dbe417bdb68cfc211 here now 🙂 

As for using another third party library, I think I might just package up the one we're using in our own `@directus/tsconfig`, just so we're gonna be able to easily make changes moving forwards (without having to add extend overrides)

Reply:
> As for using another third party library, I think I might just package up the one we're using in our own `@directus/tsconfig`, just so we're gonna be able to easily make changes moving forwards (without having to add extend overrides)

Love it ❤️

Reply:
https://github.com/directus/directus/pull/16825/commits/1795b8292990ca740b250726739364b009531b6b 🙌🏻

Reply:
Awesome 🥇